spark/src/main/java/org/opensearch/sql/spark/flint/FlintIndexMetadataReaderImpl.java

```diff
@@ -36,25 +36,25 @@ private String getIndexName(IndexDetails indexDetails) {
     if (FlintIndexType.SKIPPING.equals(indexDetails.getIndexType())) {
       return "flint"
           + "_"
-          + fullyQualifiedTableName.getDatasourceName()
+          + fullyQualifiedTableName.getDatasourceName().toLowerCase()
```

Comment:
before return, lowercase the entire string rather than each piece?

Reply:
I am doing that as well

Reply:
yup...raising that just changed.